requirements.txt

```diff
@@ -1,5 +1,5 @@
 six>=1.9.0
 boto>=2.36.0
-luigi>=1.1.2
+luigi>=1.1.2,<2.7.0
```

Comment:
@toru-takahashi this package is actually relying on `luigi < 2.5.0`. I confirmed that test passing with `luigi==2.4.0`

Reply:
Thank you. I'll fix it later.